packages/dapp/src/utils/arweave/key.json

```diff
@@ -0,0 +1 @@
REBZsSDmDz8PFUTJ_JQbq0M_XtCjKRSqqNVpAAXkX2xdyV81gphmBZ2A_Nsc33-S3O1d_DeGc_cwsrYn1wP2Wm2sUe5tVUrfcwqHELsxuDMGHTK4bDtrBrVHyqGygJUVv4zpXiDr6Cc5gHxHweRkLqd3AEddEKPv9CxAlbeQ2g3i6SCXHpPqquir35i8O94F1WMfK1P0y9lnrwLeJltIlnOAfCBF_Vfsp3MSHH1da4jyjJTyGo-vBXl2kZQqoSJZVBXYI0cvpMPfTEXiZlqXjiCd1nql5LLvwyXTkwumG3lrMrSrITHtVtA8MAYsbrxjkeM0KkkERVfKOID6jmdJ-H2eOrhnzOyhPihXpFGKBZyjkgZW60-RtCO9JlY8GOEz0Z-2Qq0hYajQia_cBzGmtRonkB4bZwgx4sHPoifw06091S8NaMYv0O9zKELRwq-yWtaLTuK5eWkCimF1jO5kxueaMid5CT0y3XL59ctwhU58","e":"AQAB","d":"kcSVOqhCkIQffDs3P_RazMpZSfl0Bg0c5QqRdV8JzEhWtwuXKGF5n0PjH8exC7EocFqrvapoVgFCPNejx3ajhdvY9XFM9rJmsojHrZlinOzn_mEZnMWK4TSLcRVvLyLlfaeWKJxsQypfOtUGRdE98HrbwcmG-6JMA6_LeEel9QsxwK-zq8u40KimHGjtaPjABjrnrphc1x6qwP4b960xWmrNFX6NT-Pc7o75kGIXGL2VTAh3x9ltpmYWhszp8adoUOHq93UbEynO_abFziqybCvq12OrlbIqA9mM6arqqPu2G27neZABma8Hubsj54VIuYK8LwQ16sRrN1PWMDEvvIJIB4C3LztLbg17zgBM_TWRteZB1ZQ1HFMilDQmkbsj-mTUhuLGKVTLMvNrytNqYqpkH-q0TozlxHqacHwBltbozGmrV9YBNvIj1-FYMTCfIUrgGnDBgaLQUcFMW6NcgEfBOhEGgwbV78HTwmbtneZRMXeVFK4SbeP90YBFyqO07FOhBJNBTdzkARFIFu83KxzUymMQKLXeGVVnIIGO9zIFw0o_ttOqyacK2yzUcLm3cgmBlwVtBNAN1XCAY4ytQYYdBnVTc1gwHRHc7MZ7pbjRJNNCdpO0p_Jk8kUS3_K3VSrm5exLZ3pJ6CA4DUefX44yTbJKkqZN8Emum5FULNE","p":"6jez1YxLJ_49r7nuoCoOUaJanBWeYEBcjP6TGg5I9Yb5Wd1EE6oZiFaVLkb_RpF-gbSZwIxVkS90sPAFa_THVmCNWkOyK46zastZQi2WSmjRlVYHozeA-zXmxal_jvnM9Ojie7ukcqzxAsdH9Tqn_ZyQoNp70zrgOSOPEvSI95I42Lw3MMY5ry_b5BCAvfKLpl7VsKvQZkbzK5wUSB6kokTloLggHICySDLMkZvxxJWKpoAnMoSuE_HrEornOqjC8G5KUhNZOmDKWhtWtDnk0HTYq_Ef1a8splfgN6yEaUGSCDMUZdAXKQzw34KXEhmlP46HCxeLALLFDUa15qIYuQ","q":"08-bKTUgojzUvE-fq2-5gWUr6UzYkSB2DdufuiWZfY2ec-vP8txsPVRsxdUwgR5nKRL-muA071_oQPZHQeYD1wKXybXPqXps0qBT7Gogw2mJVtEO4PvqEvxUvQhQ4PpHLdOgNoYgua2qZ93dW7tn89OOuVP0gNRXyF26kuk3r2Yzgk1TkZb1fbkKhd4xacMVZfI2qkrsaWpva3WC9qUfZyoW9W-1W46w8x9aOtBrUbaUsfGhX6_m3nqZnASA1rYhMfMttgJEDlcfV--dxa4y6NddQ1t-2PU3-O4fVwXMkzjD2qA41XIXVl8O1lRF3wN6o-CNNzGImUQi6elkXyZzFw","dp":"Z6e3Qu8P4-c0zaxv2xpBtAcyKpm2ucpCK3QPDhz8VQ0jFY5DLqC9JAWNiCmLmGOTpTuUQA7bX6Gs9xGClMCD58pbRHNpSrWg2cac_hjGxq-U8Y5dWvKZdpPoSdo1xcTv3AOlGDV3FflLwHsJcZI__pFnwIhw_I615RPDohPu-gIQfnrtwfw4UzvPCSP0lhvdige2crXXSSxlOgudPj2xhs8PAQ37yCF8xU8WEer-GI5GCmH09vqZP-J0soQeyWseuuFUqvY9KwTjCPXIbYKhrU_JBG92M0-PZx7E8F69Swfl5RMWF-j8FwLPJ1vYF_6Zj4M4aMRGlEyJwNJmdmeOoQ","dq":"TtBXSfQM2pGMDy4Jmb8O5ckBl-QSSZ4sj2eDfb5z6LYDmdrLG_FNDV0WxF5ZmBH6DNyWhIac691iDJoXVbwC_cRnOpU-Wi_FwnIKCFRNMKlag0VzJeBW7w2ZcmVfwvcd-T1fxioOj0jBubhin7mSKrYzy14kArQAnfvSaswA_nEnBXtEd04WtttdwQY2mMYQdki0cpVCds81-mRVgPVcRATzWgxhDVqBGVY3Ot9Gr7ciiTwB8hr_6xTvh5BdaIeIvo9gGPOq4zMj5JKywFXbd6x7dY7cGlrUovnoC3eK8dUTe4xW2YWEjYz9dl8RSLSAs-3gXgM-ctlZrHQ99vHVrQ","qi":"D3RPXpnKKN0PpOBmDFpdvUit6DUzEvFkfJnoegcjc_bzyXFKre9q14wUPFvEK6PRNuq8R0mC4ruYF9OAEXpGJHJx2f1_s4HO0U26hjGYkhbbKzEb6g16s0YMTlOJ8JSGA1uquI_jUzK3CxnlXmpxlQz7Q0rNTPVd_wKLjIsyCFFwgjkZDRu_a_9rl_hXxFQ_5E7eV8PQ9ZvFH-EHj7L7rB8CrXWwzmFOJXhH9-BfnvYvkv13WHGpQwKPrNzw1XEegvtYbluXU5FTzNMH3Y3yt0NTenbTw5DVFU1XTE6L-Jkdf53j4eiARdMDUSowU3-om3dHas-kQKNQJy07u7UEIA"},"address":"MBpZV7BMYtsU_-wqgWYoIWyedVVpXK9topTVOIKW_GA"}
```

Comment:
this is a test account - We should probably either be switching to a server framework (nextjs?) or put together a basic server that takes care of the keys and relaying transactions to arweave

Reply:
If we were able to keep our real key as an ENV variable and stay away from requiring any server, that would be first prize. We can discuss further this week as we explore more though. Happy with all these files now for testing purposes.